### PR TITLE
Em beutler

### PR DESCRIPTION
**Pull request by embeutler**
test

### CODE REVIEW

File: Splash_Index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="splash_style.css">
+  <style > 
+body{
```
Comment (jmckenney) [inline Splash_Index.html:10]: don't forget to mind the tabs/spaces and keep it formatted.

File: Splash_Style.css [hunk 2]
```diff
@@ -0,0 +1,94 @@
+@import url('https://fonts.googleapis.com/css?family=Barriecito|Thasadith&display=swap');
+ 
+body{ 
+  background-image: url("https://media1.tenor.com/images/9bc211bce15728fe101405f6f851ed52/tenor.gif?itemid=14243222");
+  background-repeat: no-repeat;
+background-size: 100%;
```
Comment (jmckenney) [inline Splash_Style.css:6]: spacing tab issue here as well.

**Comment by lyntan** [comment]
checked test pr

File: Untitled-1.txt [hunk 3]
```diff
@@ -1 +1,2 @@
-test
\ No newline at end of file
+test ]
```
Comment (jmckenney) [inline Untitled-1.txt:1]: can probably remove this file:)

**Pull request closed without merging**